src/plugins/notebook/components/NotebookEntry.vue

```diff
@@ -410,7 +410,7 @@ export default {
       this.manageEmbedLayout();
       this.timestampAndUpdate();
     },
-    convertMarkDownToHtml(text) {
+    convertMarkDownToHtml(text = '') {
```

Comment:
Added a default of empty string here as this was sometimes `undefined` and causing issues. Still not sure why